README.md

```diff
@@ -1,11 +1,11 @@
-# narendrach-research-reflection
+# Narendra's Research and Reflection Journal 
 This is my research and reflection journal for DGL 104 winter 2023
 
 
-# course goals
+## Course Goals
 
 - learn new languages that helps you in the future.
-- In time work complition and submission.
+- In time work completion and submission.
```

Comment:
I didn't quite understand this statement. I assume you meant "completion" when you wrote "complition"?